azkaban-web-server/src/main/bash/azkaban-web-shutdown.sh

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env bash
 # Shutdown script for azkaban web server
-installdir="$(dirname $0)/.."
+set -o nounset
+source "$(dirname $0)/util.sh"
 
+installdir="$(dirname $0)/.."
+maxattempt=5
```

Comment:
Shall we fail faster? If the process won't shut down within 10 seconds, I would expect a problem.   Is it better to fail the operation and let operator take some manual action sooner? At this point, the service is probably unavailable already. It's better to react fast.